app/controllers/statistics_announcements_controller.rb

```diff
@@ -1,26 +1,29 @@
 # Note that announcement pages are rendered by the `government-frontend` application.
 class StatisticsAnnouncementsController < PublicFacingController
-  include PublicDocumentRoutesHelper
-
   enable_request_formats(index: [:js])
 
   def index
-    @filter = Frontend::StatisticsAnnouncementsFilter.new(filter_params)
-    expire_cache_for_index_on_next_announcement_expiry(@filter.results)
-    if request.xhr?
-      skip_slimmer
-      render partial: "statistics_announcements/filter_results"
-    end
+    redirect_to_research_and_statistics
```

Comment:
Does this need a locale restriction?

Reply:
In this case it's not required, as there aren't other locales for statistics announcements. E.g. https://www.gov.uk/government/statistics/announcements.fr